spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -18,6 +31,12 @@
     context "on #{plat[:name]} we should behave ourselves" do
```

Comment:
@glennsarti Those statements are all over the Vanagon spec files, I don't particularly like them either but the scope to change that is much larger than this PR